src/pages/create/forms/FundingForm/DistributionSplitsSection/DistributionSplitModal.tsx

```diff
@@ -217,16 +213,22 @@ export function DistributionSplitModal({
     onClose()
   }
 
+  /**
+   * Set new distribution limit
+   */
   useEffect(() => {
     if (distributionLimitIsInfinite || !amount) return
+
+    const newAmount = parseFloat(stripCommas(amount))
```

Comment:
This line is the fix.

I'm pretty concerned that this is happening in more places than this, because `FormattedNumberInput` emits the raw string (which includes the commas). I think it should be emitting a number (after all it's a 'number input) or at very least a number string wihtout commas.